cmd/zetatool/config/config.go

```diff
@@ -0,0 +1,81 @@
+package config
```

Comment:
This seems related to the filtering feature, this should be in the deposit filter package

Reply:
I would prefer to keep it more generic in case there are more tools added. Some of the more specific fields can be passed as arguments and removed from the config file.